src/lib/api-client.js

```diff
@@ -86,6 +86,7 @@ export function checkOutItems(studentId, itemAddresses){
             Dispatcher.handleAction('OVERRIDE_REQUIRED');
         }else if(error === 'Invalid Admin'){
             Dispatcher.handleAction('CLEAR_ADMIN_CODE');
+            Dispatcher.handleAction('INVALID_CODE');
```

Comment:
Can we just use one action here? Maybe rename `CLEAR_ADMIN_CODE` to `INVALID_CODE`, and have the authentication store subscribe to that: https://github.com/TheFourFifths/consus-client/blob/dev/src/store/authentication-store.js#L48